.travis.yml

```diff
@@ -2,21 +2,16 @@ language: node_js
 sudo: false
 dist: precise
 node_js:
+    - "node"
```

Comment:
Yeah I think thats a good idea, we can do `- "8"` and also leave `- "node"` so we automatically test against latest as they update.